recipes/gstreamer.recipe

```diff
@@ -73,4 +73,7 @@ class Recipe(recipe.Recipe):
                 self.config.distro_version in [DistroVersion.UBUNTU_TRUSTY, DistroVersion.UBUNTU_UTOPIC, DistroVersion.UBUNTU_VIVID, DistroVersion.UBUNTU_WILY, DistroVersion.UBUNTU_XENIAL, DistroVersion.DEBIAN_STRETCH]):
             _patch = self.relative_path('gstreamer/0001-gst-parse-switch-to-lex-param.patch')
             shell.apply_patch(_patch, self.build_dir, 1)
+            _patch = self.relative_path('gstreamer/0002-also-check-for-winpthread.patch')
```

Comment:
we should use the patches structure instead.

Reply:
I tried but for some reason that I didn't follow the repo is hard reset to HEAD *after* the patches are applied during the prepare stage.
I suspect that this is the reason of the former 0001 patch being there and I simply added a new patch below. It works that way....

Reply:
I'm holding this until the pending parts of INF-264 are ready to review